libs/styles/styles.css

```diff
@@ -1,134 +1,587 @@
+  --s2-transparent-white-900: rgb(255 255 255 / 94%);
+  --s2-transparent-black-100: rgb(0 0 0 / 9%);
+  --s2-transparent-black-300: rgb(0 0 0 / 15%);
+  --s2-transparent-black-400: rgb(0 0 0 / 22%);
+  --s2-transparent-black-800: rgb(0 0 0 / 84%);
+  --s2-transparent-black-900: rgb(0 0 0 / 93%);
+  --s2-gray-25: rgb(255 255 255);
+  --s2-gray-50: rgb(248 248 248);
+  --s2-gray-100: rgb(233 233 233);
+  --s2-gray-300: rgb(218 218 218);
+  --s2-gray-400: rgb(198 198 198);
+  --s2-gray-500: rgb(143 143 143);
+  --s2-gray-700: rgb(80 80 80);
+  --s2-gray-800: rgb(41 41 41);
+  --s2-gray-900: rgb(19 19 19);
+  --s2-blue-800: rgb(75 117 255);
+  --s2-blue-900: rgb(59 99 251);
+  --s2-blue-1000: rgb(39 77 234);
+  --s2-red-200: rgb(255 235 232);
+  --s2-red-300: rgb(255 214 209);
+  --s2-red-600: rgb(255 118 101);
+  --s2-red-800: rgb(240 56 35);
+  --s2-red-900: rgb(215 50 32);
+  --s2-red-1000: rgb(183 40 24);
+  --s2-red-1100: rgb(156 33 19);
+  --s2-orange-600: rgb(252 125 0);
+  --s2-orange-700: rgb(232 106 0);
+  --s2-yellow-400: rgb(245 199 0);
+  --s2-yellow-600: rgb(210 149 0);
+  --s2-celery-600: rgb(93 180 31);
+  --s2-celery-700: rgb(82 161 25);
+  --s2-green-700: rgb(11 164 93);
+  --s2-green-900: rgb(5 131 78);
+  --s2-green-1000: rgb(3 110 69);
+  --s2-seafoam-700: rgb(11 162 134);
+  --s2-seafoam-900: rgb(7 129 109);
+  --s2-cyan-600: rgb(48 167 254);
+  --s2-cyan-900: rgb(11 120 179);
+  --s2-indigo-800: rgb(122 106 253);
+  --s2-indigo-900: rgb(113 85 250);
+  --s2-purple-800: rgb(166 92 231);
+  --s2-purple-900: rgb(154 71 226);
+  --s2-fuchsia-800: rgb(200 68 220);
+  --s2-fuchsia-900: rgb(181 57 200);
+  --s2-magenta-800: rgb(240 45 110);
+  --s2-magenta-900: rgb(217 35 97);
+  --s2-pink-800: rgb(228 52 163);
+  --s2-pink-900: rgb(206 42 146);
+  --s2-turquoise-800: rgb(10 141 153);
+  --s2-turquoise-900: rgb(8 126 137);
+  --s2-brown-800: rgb(154 123 77);
+  --s2-brown-900: rgb(139 109 66);
+  --s2-silver-800: rgb(128 128 128);
+  --s2-silver-900: rgb(114 114 114);
+  --s2-cinnamon-800: rgb(184 109 70);
+  --s2-cinnamon-900: rgb(170 94 56);
+  --s2-drop-shadow-color-100: rgb(0 0 0 / 12%);
+  --s2-drop-shadow-color-200: rgb(0 0 0 / 16%);
+  /* S2 Color Spec (semantic) */
+  --s2-informative-color-800: var(--s2-blue-800);
+  --s2-negative-color-200: var(--s2-red-200);
+  --s2-negative-color-300: var(--s2-red-300);
+  --s2-negative-color-600: var(--s2-red-600);
+  --s2-negative-color-800: var(--s2-red-800);
+  --s2-negative-color-900: var(--s2-red-900);
+  --s2-negative-color-1000: var(--s2-red-1000);
+  --s2-negative-color-1100: var(--s2-red-1100);
+  --s2-notice-color-700: var(--s2-orange-700);
+  --s2-positive-color-700: var(--s2-green-700);
+  --s2-positive-color-900: var(--s2-green-900);
+  --s2-positive-color-1000: var(--s2-green-1000);
+  --s2-accent-color-900: var(--s2-blue-900);
+  --s2-accent-color-1000: var(--s2-blue-1000);
+
+  /*** Milo Semantic Colors ***/
+  --link-color: var(--color-blue-900, #035fe6);
```

Comment:
We should take the opportunity to make all color declarations consistent, either uppercase or lowercase.

Reply:
I believe they are currently all lowercase, let me know if I missed anything?

Reply:
I noticed some upper case values
https://github.com/adobecom/milo/pull/2484/files#diff-c685e2a5191e8c907b9958b7f267d04cc48fdce1491da2c38461955f5d2bd913R19
https://github.com/adobecom/milo/pull/2484/files#diff-c685e2a5191e8c907b9958b7f267d04cc48fdce1491da2c38461955f5d2bd913R20-R23

Reply:
Ahh just realized you were referencing the hex values, while I was looking for color variable names. These should be updated now👍